src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -417,4 +418,25 @@ protected function compileEndPushIf()
     {
         return '<?php $__env->stopPush(); endif; ?>';
     }
+
+    /**
+     * Compile conditional HTML attributes into valid PHP.
+     *
+     * @param  string  $expression
+     * @return string
+     *
+     * @throws \Illuminate\Contracts\View\ViewCompilationException
+     */
+    protected function compileMaybe($expression)
+    {
+        $parts = explode(',', $this->stripParentheses($expression), 2);
+
+        if (2 !== count($parts)) {
+            throw new ViewCompilationException('The @maybe directive requires exactly 2 parameters.');
+        }
+
+        [$attribute, $data] = array_map('trim', $parts);
+
+        return "<?php if($data !== '' && $data !== null && trim(is_bool($data) ? ($data ? 'true' : 'false') : $data) !== '') echo $attribute . '=\"' . e(is_bool($data) ? ($data ? 'true' : 'false') : $data) . '\"'; ?>";
```

Comment:
I guess the `filled()` helper fits well the requirements.

```php
return "<?php if(filled($data)) echo $attribute . '=\"' . is_bool($data) ? ($data ? 'true' : 'false') : e($data) . '\"'; ?>";
```

Reference: https://laravel.com/docs/12.x/helpers#method-filled

Reply:
Thanks, Rodrigo. Yes, you are right. But I prefer to not add to the call stack here. Marking is resolved and will see what Taylor thinks. 🙏